packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -137,25 +132,45 @@ class BottomNavigationBar extends StatefulWidget {
   /// If [type] is null then [BottomNavigationBarType.fixed] is used when there
   /// are two or three [items], [BottomNavigationBarType.shifting] otherwise.
   ///
-  /// If [fixedColor] is null then the theme's primary color,
-  /// [ThemeData.primaryColor], is used. However if [BottomNavigationBar.type] is
-  /// [BottomNavigationBarType.shifting] then [fixedColor] is ignored.
+  /// Prefer [selectedItemColor] over [fixedColor], since [selectedItemColor]
+  /// applies to both shifting and fixed selected item colors. If
+  /// [selectedItemColor] is null, [fixedColor] will be used for fixed
+  /// [BottomNavigationBar]s. If [selectedItemColor] and [fixedColor] are
+  /// null, then the theme's primary color, [ThemeData.primaryColor] is used.
+  ///
+  /// [iconSize] must be at least 8.0 for the [Icon]s to display properly.
```

Comment:
Why do they need to be 8?

Reply:
To be honest, I don't exactly know why. Something to do with `BoxConstraints` somewhere in the tree. All I know that if the `iconSize` is less than 8, this happens: 
<img width="250" alt="screen shot 2019-02-12 at 10 31 45 pm" src="https://user-images.githubusercontent.com/4229329/52722539-d0138800-2f79-11e9-86f1-4ab48295aea0.png">

But maybe I should remove my assert because perhaps this is more informative to users?

Reply:
Let Hans decide.

Reply:
We should figure out the reason and include it in the documentation.

BTW, nit: see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-correct-grammar (avoid starting sentences with a lower case letter)